src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/SinglePhasePoromechanicsEFEM.hpp

```diff
@@ -341,7 +341,8 @@ struct StateUpdateKernel
           arrayView1d< real64 > const & aperture,
           arrayView1d< real64 const > const & oldHydraulicAperture,
           arrayView1d< real64 > const & hydraulicAperture,
-          arrayView2d< real64 > const & fractureEffectiveTraction )
+          arrayView2d< real64 > const & fractureEffectiveTraction,
+          arrayView1d< integer > const & fractureState )
```

Comment:
```suggestion
          arrayView1d< FractureState > const & fractureState )
```

Reply:
keep it as integer to avoid compilation error